rebasehelper/patch_helper.py

```diff
@@ -232,7 +229,7 @@ def _git_rebase(cls):
                     ret_code = 0
             else:
                 break
-        deleted_patches = cls._update_deleted_patches(deleted_patches, inapplicable_patches)
+        deleted_patches = cls._update_deleted_patches(inapplicable_patches)
```

Comment:
Previous occurence of `deleted_patches` is unused after this change, you should remove it.